hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -276,14 +275,8 @@ void initClusterAndEnv() throws IOException, InterruptedException, TimeoutExcept
             -> (RootedOzoneFileSystem) FileSystem.get(conf));
 
     if (useOnlyCache) {
-      if (omRatisEnabled) {
```

Comment:
here we are removing test scenario, where ratis is enabled, but double buffer is not flushed, and entire operation works over cache only. This is to catch issues when double buffer is taking to flush.
Please check if this test scenario can be enabled back?
The testcase is blocking as daemon is shutdown in this case, this test cases fixed with,

https://github.com/apache/ozone/pull/5857/commits/493323a2632a5f28f0486f775e50c1817de941b1

Reply:
Good catch!  Let's revert this change.